pkg/controller/manager/manager_controller_test.go

```diff
@@ -200,18 +196,14 @@ var _ = Describe("Manager controller tests", func() {
 		})
 
 		It("should reconcile if user supplied a manager TLS cert", func() {
-			// Create a manager cert secret with invalid DNS name
+			// Create a manager cert secret.
 			dnsNames := []string{"manager.example.com", "192.168.10.22"}
+			testCA := test.MakeTestCA("manager-test")
 			secret, err := render.CreateOperatorTLSSecret(
-				nil, render.ManagerTLSSecretName, render.ManagerSecretKeyName, render.ManagerSecretCertName, render.DefaultCertificateDuration, nil, dnsNames...)
+				testCA, render.ManagerTLSSecretName, render.ManagerSecretKeyName, render.ManagerSecretCertName, render.DefaultCertificateDuration, nil, dnsNames...)
```

Comment:
I think you need to save this secret as `userSecret` and we need a check at the end of this test that checks  `secret.Data` equals `userSecret.Data`. That should verify that the user provided secret doesn't get overwritten.